server/graphql-schema.js

```diff
@@ -973,9 +994,23 @@ const graphqlSchema = gql`
     }
 
     type Mutation {
+        """
+        Get the available mutations for the given AT.
+        """
         at(id: ID!): AtOperations!
+        """
+        Get the available mutations for the given AT Version.
+        """
         atVersion(id: ID!): AtVersionOperations!
         """
+        Get the available mutations for the given Browser.
```

Comment:
Browser isn't capitalized 😝 sorry I have to play my former-English-teacher card.